### PR TITLE
Mention offset arg. in Enum.with_index/2 and Stream.with_index/2 docs

### PR DESCRIPTION
**Pull request by eksperimental**

### CODE REVIEW

**Comment by eksperimental** [comment]
coudln't come up with any better way to put it.

other alternatives are:
> If `offset` is given, it will be added to the indexes of each element.

**Comment by josevalim** [comment]
Why if we say: "If an `offset` is given, we will index from the given offset instead of zero."?

**Comment by eksperimental** [comment]
@josevalim do you think this is clear enough for when offset is negative?

**Comment by josevalim** [comment]
I think so. For example, we will start counting from -3. Which means -3, -2, -1, 0, 1, 2, 3... What do you think?

File: lib/elixir/lib/enum.ex [hunk 1]
```diff
@@ -2443,6 +2443,8 @@ defmodule Enum do
   Returns the enumerable with each element wrapped in a tuple
   alongside its index.
 
+  If an `offset` is given, we will index from the given offset instead of from zero.
```
Comment (whatyouhide) [inline lib/elixir/lib/enum.ex]: Maybe "[...] offset instead of zero" instead of "[...] given offset instead of **from** zero"? Not sure. 😕

**Comment by eksperimental** [comment]
yes. that's clear enough. I just changed your sentence to read "...instead of **from** zero"

File: lib/elixir/lib/enum.ex (hunk 1 above)
Comment (eksperimental) [inline lib/elixir/lib/enum.ex]: a native speaker would come in handy

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**